.github/workflows/agw-workflow.yml

```diff
@@ -0,0 +1,301 @@
+---
+name: agw-workflow
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+    paths:
+      - 'orc8r/**'
+      - 'lte/**'
+  pull_request:
+    branches:
+      - master
+jobs:
+  lte-test:
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8.5'
+      - name: Install python3-aioeventlet from the magma apt repo
+        run: |
+            cd nms/app
+            # Install python3-aioeventlet from the magma apt repo
+            cat ${{ env.MAGMA_ROOT }}/orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub | sudo apt-key add -
+            echo "deb https://facebookconnectivity.jfrog.io/artifactory/list/dev-focal/ focal main" | sudo tee /etc/apt/sources.list.d/fbc.list
```

Comment:
let's use the artifactory.magmacore.org instead

Reply:
Sure!.. can you confirm the URL.. is it https://artifactory.magmacore.org:443/artifactory/debian/focal?